matching-notification-api/src/main/resources/ApplicationResources.properties

```diff
@@ -2,9 +2,7 @@ phenotips.matching.ajaxutils.requestSent=Sending request...
 phenotips.matching.ajaxutils.done=Done
```

Comment:
I don't see how this commit is related to VS, all changes seem to be relevant for matching notification and exomiser (both already present without VS), not VS. Commit title should be adjusted to reflect that